src/subscription/__tests__/subscribe-test.js

```diff
@@ -545,22 +545,22 @@ describe('Subscribe', () => {
     });
   });
 
-  it('invalid query should result in error', async () => {
-    const invalidAST = parse(`
+  it('unknown field should result in closed subscription', async () => {
+    const ast = parse(`
       subscription {
-        invalidField
+        unknownField
       }
     `);
 
-    expect(() => {
-      subscribe(
-        emailSchema,
-        invalidAST,
-      );
-    }).to.throw('This subscription is not defined by the schema.');
+    const pubsub = new EventEmitter();
+
+    const { subscription } = createSubscription(pubsub, emailSchema, ast);
+
+    const payload = await subscription.next();
+    expect(payload).to.deep.equal({ done: true, value: undefined });
```

Comment:
So we won't actually get to this step because `subscription { unknownField }` would fail validation and return an informative error?